docs/01_setup.md

```diff
@@ -148,6 +153,29 @@ https://vercel.com/new
   <img width="45%" src="./images/vercel_configure.png" />
 </div>
 
+## Poll finalization
+
+Once the voting time has ended, as a coordinator, first you need to merge signups and messages (votes). Head to MACI contracts repository and run the merge command with the deployed poll:
+
+```bash
+cd contracts  && \
+pnpm merge:[network] --poll [poll-id]
+```
+
+> [!IMPORTANT]
+> If you use the same MACI contract, each time you deploy a poll the id increases by one.
```

Comment:
something to keep in mind is that for 1.2.2 you cannot use the same maci contract for more than one poll, as if one user signups after the first poll has been finalized, the second poll won't be able to be finalized (this works now in the latest dev)

Reply:
also wanna leave a question here: I think we wanna let the recent qf rounds (eth mexico?) to use v1.2.2, then how should we update the document? to a certain branch? since the latest branch is based on the latest version of maci.

Reply:
You need to finalize it before starting a new poll or you can't use the same MACI contract?

Reply:
short story, you can't. if another user signs up after the trees are merged, then there is a skew in the state index which causes a mismatch with the local code